### PR TITLE
Update app style

### PR DESCRIPTION
**Pull request by J-C-S-V**
Hello dear reviewer 👋, I hope you're having a nice day today 😊

This is what I did on this branch:

- Style the app according to the Zeplin's design
- Keep functionality of the app

Best regards! ✨

### CODE REVIEW

File: src/styles/BookCard.scss [hunk 1]
```diff
@@ -52,8 +53,63 @@
...
+.book__right-h3-2 {
+  font-weight: 100;
+}
+
+.book__right > * {
+  margin: 0.5rem 0;
+}
+
+.book__left-h2-2 {
+  font-size: 1.5rem;
```
Comment (Qoosim) [inline src/styles/BookCard.scss:114]: - [x] Great work implementing your design in line with the project requirements. However, there're few things you need to work on before your project could be approved. Kindly ensure that you produce a close replica of the Zeplin design. This includes background color, text-color, margin, padding among others. Look at the both screenshots below, check the areas I marked with red lines and walk your desgin alongside the Zeplin design. Trust me, you're almost there.

### Your Design
your-design

### Zeplin Design 
temp-design

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @J-C-S-V, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Gitflow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by J-C-S-V** [comment]
Hello @Qoosim, 👋

Thanks for the feedback, really appreciate it.

Regards!

**Review by binhussen — APPROVED** [review]
Hi @J-C-S-V , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Used React-redux ✔️
- Sticked Zapline Design ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by J-C-S-V** [comment]
Thank you @binhussen !!!!!!!!

**Pull request merged**